db_migrations/1.02/tracking.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE user_tracking_timing (
+    id int PRIMARY KEY,
+    page_name text,
+    recorded_at text,
```

Comment:
Let's add default values so it looks better. And initialize utf8 encoding so nothing weird gets entered.

CREATE TABLE `user_tracking_timing` (
  `id` int unsigned NOT NULL AUTO_INCREMENT,
  `page_name` text NOT NULL DEFAULT '',
  `recorded_at` datetime DEFAULT NULL,
  `city` text NOT NULL DEFAULT '',
  `country` text NOT NULL DEFAULT '',
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;